cmd/sqlflow/main.go

```diff
@@ -42,7 +42,8 @@ func compile(cgName, sqlProgram, datasource string) (string, error) {
 		}
 		sess := sql.MakeSessionFromEnv()
 		sess.DbConnStr = datasource
-		return couler.GenCode(spIRs, sess)
+		cg := &couler.Codegen{}
```

Comment:
Giving that `workflow.New("argo")` returns a `workflow/codegen/couler.Codegen`, I think we might want users to call `workflow.New("argo")`